docs/book/element/phone-number.md

```diff
@@ -0,0 +1,94 @@
+# PhoneNumber
+
+`Laminas\I18n\PhoneNumber\Form\Element\PhoneNumber` is a text input form element that automatically attaches a PhoneNumber validator.
+
+## Basic Usage
+
+This element automatically adds a `type` attribute of value `tel`.
+
+```php
+use Laminas\I18n\PhoneNumber\Form\Element\PhoneNumber;
+use Laminas\Form\Form;
+
+$phone = new PhoneNumber('number');
+$phone->setLabel('Phone Number');
+$phone->setAttributes([
+    'autocomplete' => 'tel', // 'tel-national' is also an option.
+]);
+
+$form = new Form();
+$form->add($phone);
+```
+
+Using array notation:
+
+```php
+use Laminas\I18n\PhoneNumber\Form\Element\PhoneNumber;
+use Laminas\Form\Form;
+
+$form = new Form();
+$form->add([
+    'type' => PhoneNumber::class,
+    'name' => 'phone-number',
+    'options' => [
+        'label' => 'Phone Number',
+    ],
+    'attributes' => [
+        'autocomplete' => 'tel'
+    ],
+]);
+```
+
+TIP: **Autocomplete Attribute**
+To make it easier for users to autofill valid information, the [autocomplete attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete) can be used. A value of `tel` should autofill the users phone number with the country dialling code, whereas a value of `tel-national` will autofill the number without the country dialling code.
+
+## Available Options
```

Comment:
I am sorry, because I forgot to say that the current page structure of the form elements is bad and should not be used.